### DISCUSSION TITLE
Microsoft Store cooperates with Tencent App Store.  Is WSA resurrected in sight?

### DISCUSSION THREAD

**Discussion by lisyclover** [microsoft/WSA#561] [General]
Microsoft Store cooperates with Tencent App Store.  Is WSA resurrected in sight?
https://mspoweruser.com/windows-subsystem-for-android-wsa-might-see-continued-life-at-least-in-china/

https://news.microsoft.com/zh-cn/%E8%85%BE%E8%AE%AF%E5%BA%94%E7%94%A8%E5%AE%9D%E4%B8%8E-microsoft-store-%E8%BE%BE%E6%88%90%E5%90%88%E4%BD%9C%EF%BC%8Cwindows-%E5%8F%AF%E7%9B%B4%E6%8E%A5%E8%BF%90%E8%A1%8C%E7%A7%BB%E5%8A%A8%E5%BA%94/

**Comment by XDRSZYHRL** [comment]
![image](https://github.com/microsoft/WSA/assets/59906163/9860487d-eb47-4348-b492-bebcfd1e0ac7)

**Reply by tony-sung** [reply]
现在已经改成aow原生方案了

**Comment by Slion** [comment]
Thanks for sharing that's very encouraging.

**Comment by 1115714829** [comment]
现在的版本还不是WSA 现在安装就提示让关闭Hyper-V，和其他模拟器一样。

**Reply by lisyclover** [reply]
现在已经基于hyper v了

**Comment by Wduran84** [comment]
check out https://windowsxlite.com/downloads/ it has a version of WSA that uses Google Play Store
